script.js

```diff
@@ -0,0 +1,72 @@
+const addBtn = document.getElementById('add');
+// Declaring varibale for the books' container
+const main = document.querySelector('main');
+
+// Creating Array of objects(Book List)
+let booksList = [
+  {
+    id: 1,
+    title: 'The Alchemist',
+    author: 'Paulo Coelho',
+  },
+
+  {
+    id: 2,
+    title: 'The Animal farm',
+    author: 'Paulo Jack',
+  },
+];
```

Comment:
Great job team 🤝

- [ ] Kindly remove static objects and make sure the array is populated dynamically by appending the to the array. You may please revisit the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md) to get a better understanding of what is required.
- [ ] Kindly take a look at the screenshot below and compare it with what was given and then work around it to improve your UI.
- [ ] Your Project after clicking the add button a couple of times.
![image](https://user-images.githubusercontent.com/63917725/197695582-3eaa3409-92f5-4c2c-b05f-5c50e5883369.PNG)